### PR TITLE
giti 1.0 (new formula)

### PR DESCRIPTION
**Pull request by kevcooper**
giti is a script that speeds up the process of initializing new git repositories

### CODE REVIEW

File: Library/Formula/giti.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  url "https://github.com/kevcooper/giti/archive/v1.0.tar.gz"
+  head "https://github.com/kevcooper/giti.git"
+  sha256 "24216a3ce4df46f8e37455a474afd4dc3bd2cf1e78d1c942a63119044c6c5e14"
+
+  def install
+    bin.install "giti"
+  end
+
+  test do
+    system "#{bin}/giti", "--help"
```
Comment (dunn) [inline Library/Formula/giti.rb]: Any chance of a test that checks more of the functionality? Git is available on every system so you can make a test repository.

**Comment by dunn** [comment]
We can't include every piece of software, so very new projects and self-submissions are rarely accepted, unfortunately.

**Comment by kevcooper** [comment]
Yeah I'll add some real tests :) and sorry about adding to the clutter if this is too basic for a PR

**Comment by DomT4** [comment]
Yeah, It's failing two of our standard audit checks on this subject:

```
 * GitHub repository not notable enough (<10 forks, <10 watchers and <20 stars)
 * GitHub repository too new (<30 days old)
```

For now, you may wish to maintain it in your own tap, which is very easy to set up and would allow users to use Homebrew to install it still, just via `brew install kevcooper/giti/giti` instead of `brew install giti`.

Closing here per the above, but thanks for the submission and when the tool gains a little age & popularity you're welcome to resubmit the PR. Thanks!

**Pull request closed without merging**